src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Noticia+Text:ital,wght@0,700;1,400;1,700&family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  font-size: 12px;
+}
```

Comment:
Be consistent. If you set font styles, set them in one place.
Move `font-size` under `html` tag.